spm/internal/spm_repositories.bzl

```diff
@@ -635,6 +671,12 @@ spm_repositories = repository_rule(
             (e.g. .macOS(.v10_15))\
             """,
         ),
+        "workspace_file": attr.label(
+            doc = """\
+            If using any local packages, this attribute needs to be set to the value \
+            `//:WORKSPACE`.\
+            """,
```

Comment:
Would it make sense to set this up with a default value? Like:
```
        "_workspace_file": attr.label(
             default = "//:WORKSPACE",
             doc = """\
             If using any local packages, this attribute needs to be set to the value \
             `//:WORKSPACE`.\
             """,
    ),
```

Reply:
I did not know about the magic of the unnamed `@` at the front of the label. That worked perfectly. I am putting a PR together, right now. Thanks.

Reply:
PR #65.